nursery/hash-data-using-whirlpool.yml

```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: hash data using Whirlpool
+    namespace: data-manipulation/hashing/whirlpool
+    authors:
+      - william.ballenthin@mandiant.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
```

Comment:
Is `Encryption-Standard Algorithm` correct here?

Reply:
good catch